### PR TITLE
mod_glossary

### PR DESCRIPTION
**Pull request by cliffcaseyyet**
**Description**
Added/modified statements for:
- Comment Created
- Comment Deleted
- Entry Viewed

### CODE REVIEW

File: src/transformer/utils/get_activity/glossary_comment.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * @param \stdClass $course The course object.
+ * @param \stdClass $comment The glossary entry comment object.
+ * @return array
+ */
+function glossary_comment(array $config, \stdClass $course, \stdClass $comment) {
+    $repo = $config['repo'];
+    $entry = $repo->read_record_by_id('glossary_entries', $comment->itemid);
+    
+    $courselang = utils\get_course_lang($course);
+    $commenturl = $config['app_url'].'/mod/glossary/showentry.php?eid='.$entry->id.'#c'.$comment->id;
```
Comment (milt) [inline src/transformer/utils/get_activity/glossary_comment.php:42]: Am I being a herb by putting spaces around the dots when strcatting like this? Without them it starts to look like an esolang to me

File: tests/mod_glossary/comment_created/comment_created_test.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     * @return string
+     */
+    protected function get_plugin_name() {
+        return "glossary";
+    }
+
+    /**
+     * Appease auto-detecting of test cases. xapi_test_case has default test cases.
+     *
+     * @covers ::discussion_viewed
```
Comment (milt) [inline tests/mod_glossary/comment_created/comment_created_test.php]: This annotation is wrong (in all three tests). I only recently noticed it myself and have probably also pasta'd the wrong thing in there. No idea what it does.

**Review by milt — CHANGES_REQUESTED** [review]
Great work. Correct (or maybe remove?) the test `@covers` annotation and it's good to go.

File: tests/mod_glossary/comment_created/comment_created_test.php (hunk 2 above)
Comment (cliffcaseyyet) [reply tests/mod_glossary/comment_created/comment_created_test.php]: done

File: src/transformer/utils/get_activity/glossary_comment.php (hunk 1 above)
Comment (cliffcaseyyet) [reply src/transformer/utils/get_activity/glossary_comment.php:42]: yes.

**Review by milt — APPROVED** [review]

**Pull request merged**